lib/improver/utilities/pad_spatial.py

```diff
@@ -262,15 +262,15 @@ def remove_cube_halo(cube, halo_radius):
         result (iris.cube.Cube):
             New cube with the halo removed.
     """
-    halo_size_x, halo_size_y = convert_distance_into_number_of_grid_cells(
+    halo_size = convert_distance_into_number_of_grid_cells(
```

Comment:
We would need to maintain equivalent behaviour here as well as in the halo adding step.